### PR TITLE
fix: policies and env variables

### PR DESCRIPTION
**Pull request by CalvinRodo**
- rename ENV to ENVIRONMENT
- add missing policies

### CODE REVIEW

File: aws/etl_lambdas/policies.tf [hunk 1]
```diff
@@ -56,21 +56,23 @@ data "aws_iam_policy_document" "etl_policies" {
 
   statement {
 
-    effect = "Allow"
+    effect = "Allow lambdas to write to logs"
```
Comment (patheard) [inline aws/etl_lambdas/policies.tf:59]: This should be `effect="Allow"` and then you could add a `sid="Allow lambdas to write to logs"`.

**Review by patheard — CHANGES_REQUESTED** [review]
I'll work on #15 to address the pipeline issue.

File: aws/etl_lambdas/policies.tf (hunk 1 above)
Comment (CalvinRodo) [reply aws/etl_lambdas/policies.tf:59]: nice catch, I thought I had not finished the sid...

Comment (CalvinRodo) [reply aws/etl_lambdas/policies.tf:59]: Also another one to add to OPA checks

Comment (CalvinRodo) [reply aws/etl_lambdas/policies.tf:59]: Updated issue https://github.com/cds-snc/site-reliability-engineering/issues/127

**Review by patheard — APPROVED** [review]
🚀 🚢

**Pull request merged**